.gitignore

```diff
@@ -67,6 +67,8 @@ yarn-error.log
 .pnp.js
 # Yarn Integrity file
 .yarn-integrity
+.yarnrc.yml
+.yarn/
```

Comment:
Was this intentional? Any chance we could remove?

Reply:
Hi @wackerow  Thanks for the review. I have removed the files, but have a question regarding ```.yarn/``` .
When I run ```yarn``` on my local, a bunch of ```.yarn/cache/...``` files are generated. So I added the directory on ```.gitignore``` as I didn't see any past commits on these files. Can you suggest a way to get around this?

Reply:
@okdonga I see, not sure why that is happening. @samajammin have you ever had `.yarn/` generated from running `yarn`? Not necessarily against adding it to `.gitignore` just want to make sure there aren't issues with setting up the local environment for some.

Reply:
@okdonga - looks like you were able to resolve this? I don't see .gitignore in the changes anymore.

Reply:
@wackerow , @samajammin 
I think it's due to the **version of yarn** I was using. 
I was using ```2.0.0-rc.27```, and I downgraded to  ```1.22.1``` and I don't see them anymore.  Can I ask what version of yarn you are using? (or suggest using?)

Reply:
@okdonga I'm using `1.22.10`... the `-rc` is a release candidate version, not on that yet personally